doc/source/architecture.rst

```diff
@@ -60,10 +60,14 @@ It consists of the following:
     that collects general system metrics.
     For more information, see https://github.com/prometheus/node_exporter.
 
-  * ``mysql_exporter`` is a prometheus exporter
+  * ``mysql_exporter`` is a Prometheus exporter
     that collects MySQL server metrics.
     For more information, see https://github.com/prometheus/mysqld_exporter.
 
+  * ``mongodb_exporter`` is a Prometheus exporter
+    that collects MongoDB server metrics.
+    For more information, see https://github.com/dcu/mongodb_exporter.
```

Comment:
Not that link. We employ our own exporter based on this `dcu` one.
The correct link is https://github.com/Percona-Lab/prometheus_mongodb_exporter